### DISCUSSION TITLE
Formily设计器试玩地址

### DISCUSSION THREAD

**Discussion by janryWang** [alibaba/formily#1572] [Ideas]
https://designable-antd.formilyjs.org/

**Comment by MatriXiao88** [comment]
打开有点慢，不知道能不能来个国内的站点

**Comment by xintao222** [comment]
请问这个通过源码怎么运行designable-antd呢？

**Reply by javahuang** [reply]
```bash
git clone https://github.com/alibaba/designable.git --depth=1 && cd designable && npm run bootstrap && npm run start:playground
```

**Reply by xintao222** [reply]
感谢回复，我想跑 https://github.com/alibaba/formily/tree/formily_next/designable/antd这个下面的源码，运行报错了，页面白屏，不知道是否有特别的运行要求啊？

**Reply by janryWang** [reply]
```
yarn install
yarn build
yarn workspace @formily/designable-antd start
```

**Reply by xintao222** [reply]
感谢！🌹🌹🌹

**Reply by wangjie19910615** [reply]
请问怎么单独下载https://designable-antd.formilyjs.org/这个工程，我下载了https://github.com/alibaba/designable.git，这个不是jsonSchema格式，也没有事件，属性配置

**Reply by knight-peter** [reply]
_哇塞，完成度已经很高了。大佬是计划做成用户对代码完全无感知的那种程度吗？接下来还有哪些开发计划，什么时候发布到npm上。

**Reply by janryWang** [reply]
> _哇塞，完成度已经很高了。大佬是计划做成用户对代码完全无感知的那种程度吗？接下来还有哪些开发计划，什么时候发布到npm上。

这是个低代码产品，不搞无代码，8月中心发测试版

**Comment by zlren** [comment]
这个设计器本身能作为组件被业务方集成吗？
另外设计器生成的 SCHEMA 只能被 formily V2 渲染吗？formily V1 可以吗

**Reply by janryWang** [reply]
1. 设计器最终是作为npm包给用户接入的
2. 只支持2.0

**Reply by diaomina** [reply]
> 1. 设计器最终是作为 npm 包给用户接入的
> 2. 只支持2.0

请问设计器可以采用非npm包的形式接入嘛，直接通过组件之类的形式被集成。

**Comment by daifuyang** [comment]
我想知道怎么实现一个自由拖拽布局
